### DISCUSSION TITLE
What happens to my account when I die? Can I do something about it in advance?

### DISCUSSION THREAD

**Discussion by hansdejongehv** [community/community#23771] [New to GitHub]
<ol>
<li>
<p>What happens to my account when I die?</p>
</li>
<li>
<p>Will it be inherited by my heirs? Will Github let them take control based on a declaration by a notary that I died and that they are the heirs?</p>
</li>
<li>
<p>Is it OK if I provide them my username and password as part of my heritage and is it then OK that they continue to use the acccount or close it as they see fit?</p>
</li>
<li>
<p>Is there a difference in the above if I have a paid or a free account?</p>
</li>
</ol>

**Comment by AndreaGriffiths11** [comment, marked as answer]
<p>Hi <a class="mention" href="/hansdejongehv">@hansdejongehv</a>,</p>
<p>Thank you for being here. In the event that a GitHub user passes away, we can work with an authorized individual to determine what happens to the account’s content. You can invite someone to manage your user owned repositories if you are not able to. Please see <a href="https://help.github.com/en/github/setting-up-and-managing-your-github-user-account/maintaining-ownership-continuity-of-your-user-accounts-repositories" rel="nofollow noopener">https://help.github.com/en/github/setting-up-and-managing-your-github-user-account/maintaining-ownership-continuity-of-your-user-accounts-repositories</a> for more information about succession planning.</p>

**Reply by ped7g** [reply]
What about a way to mark user as passed away, so anyone visiting their profile will be aware there's no chance to further contact them?
Was this ever discussed and is there some official policy/decision?

I'm not even sure if really want this feature, I understand this may be somewhat leaking private info, then again in my life and age I was already contacted several times about particular ex-friend/ex-coworker if I can provide contact to them, because the other party was unaware they are already gone, and with the passing time I think this will become just more frequent.

In my personal case (when I die), I would definitely prefer to have some tag on my user, so people are aware, at least I'm sure about that. :)

**Comment by RichardChambers** [comment]
<p>The GitHub Deceased User Policy is at this link. <a href="https://help.github.com/en/github/site-policy/github-deceased-user-policy" rel="nofollow noopener">https://help.github.com/en/github/site-policy/github-deceased-user-policy</a></p>

**Comment by izmyname** [comment]
What if I have no successors? Will the account stay, or it'll be deleted, eventually?
